docs/api/en/textures/Texture.html

```diff
@@ -191,7 +191,7 @@ <h3>[property:number rotation]</h3>
 
 		<h3>[property:Vector2 center]</h3>
 		<p>
-			The point around which rotation occurs. A value of (0.5, 0.5) corresponds
+			The origin point around which the texture will be rotated or scaled. A value of (0.5, 0.5) corresponds
```

Comment:
The new text is somewhat confusing since there is no `scale` property on `Texture` level.

The description only mentioned rotation because of the respective `Texture.rotation` .